cookbooks/convection_box_3d/doc/convection_box_3d.md

```diff
@@ -2,30 +2,30 @@
 # Convection in a 3d box
 
 The world is not two-dimensional. While the previous section introduced a
-number of the knobs one can play with with , things only really become
+number of the knobs one can play with with, things only really become
 interesting once one goes to 3d. The setup from the previous section is easily
 adjusted to this and in the following, let us walk through some of the changes
 we have to consider when going from 2d to 3d. The full input file that
 contains these modifications and that was used for the simulations we will
-show subsequently can be found at [cookbooks/convection_box_3d.prm][].
+show subsequently can be found at `cookbooks/convection_box_3d.prm`.
 
 The first set of changes has to do with the geometry: it is three-dimensional,
 and we will have to address the fact that a box in 3d has 6 sides, not the 4
 we had previously. The documentation of the &ldquo;box&rdquo; geometry (see
-Section&nbsp;[\[parameters:Geometry_20model\]][1]) states that these sides are
-numbered as follows: &ldquo;*in 3d, boundary indicators 0 through 5 indicate
-left, right, front, back, bottom and top boundaries*.&rdquo; Recalling that we
+Section {ref}`sec:parameters:Geometry_20model`) states that these sides are
```

Comment:
I know this is not quite intuitive, but this is what the parameter labels look like right now:
```suggestion
Section {ref}`parameters:Geometry_20model`) states that these sides are
```